WordPress/src/test/java/org/wordpress/android/ui/stats/refresh/lists/sections/BaseStatsUseCaseTest.kt

```diff
@@ -55,19 +55,19 @@ class BaseStatsUseCaseTest : BaseUnitTest() {
         block.fetch(false, false)
         advanceUntilIdle()
 
-        assertData(0, localData)
+        assertData(0, remoteData)
```

Comment:
We should also update the test function name to `on fetch loads data from remote when current value is null`